### ISSUE TITLE
OPTIONAL Symbol is not visible for outside modules

### ISSUE THREAD

**Issue by abdala** [fgnass/type-assurance#5]
Error: 

```sh
Exported variable 'ComplexType' has or is using name 'OPTIONAL' from external module "type-assurance/src/index" but cannot be named.
```
I don't understand why Typescript is not being able to resolve Symbol, but if the implementation is changed to use literal or ENUM it does work.

```typescript
const OPTIONAL = "optional" as const;
// or
enum Property {
  OPTIONAL = 'optional',
}
```
Reproduce:

```typescript
// /src/test/types.ts
import {
  optional,
} from "../index.js";

//---error---v
export const ComplexType = {
  a: String,
  b: optional(String)
}
```

**Comment by fgnass** [comment]
I just tried this in a fresh project but can't reproduce that error. Can you provide a whole repo (including tsconfig and stuff)?

**Comment by abdala** [comment]
I created that sample file inside this current project, at `/src/test/types.ts`

But this is also happening inside my current project: tsconfig.json: 

```json
{
  "compilerOptions": {
    "moduleResolution": "node",
    "target": "ES2022",
    "module": "ES2022",
    "declaration": true,
    "strict": true,
    "alwaysStrict": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "strictFunctionTypes": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "esModuleInterop": true
  }
}
```

**Comment by fgnass** [comment]
I see, this happens because of `"declaration": true` in the tsconfig. I guess not using Symbols for this is the best we can do to support this use case. I'll take a look at it!

**Comment by fgnass** [comment]
I published the fix as type-assurance@1.6.2
